.gitignore

```diff
@@ -26,6 +26,25 @@ none
 # Minified JS
 /assets/js/admin/*.min.js
 /assets/js/frontend/*.min.js
+/assets/js/accounting/*.min.js
```

Comment:
Thanks, this looks much needed, what do you think about ignoring all min files by using something like `assets/**/*-min.js`

Reply:
Yes I thought about that at first however I then see this https://github.com/woocommerce/woocommerce/blob/master/.gitignore#L27-L28

So I was just following the existing pattern. I thought maybe we wanted to be verbose here. But I am ok to change it.